pkg/concurrency/dispatcher_test.go

```diff
@@ -69,18 +71,16 @@ func ExampleNewDispatcher() {
 
 // Create a new dispatcher with the name "foo".
 // Create an instance of MockWork and submit that to the dispatcher.
-func ExampleNewDispatcher2() {
-	dispatcher := concurrency.NewDispatcher("foo", 10,  5)
+func AnotherExampleNewDispatcher() {
```

Comment:
I think this needs to start with Example for go to treat it as an executable 'Example'

Reply:
You learn something new every day... I didn't realise Example tests were treated differently.

Named according to https://blog.golang.org/examples and everything passes.